modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,769 @@
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  graphdb.tigergraph.com/v1alpha1
+Kind:         TigerGraphBackupSchedule
+Metadata:
+Creation Timestamp:  2022-12-20T02:40:10Z
+Generation:          1
+Resource Version:  1696649
+UID:               f8c95418-bcb3-495b-b5e4-5083789ce11a
+Spec:
+Backup Template:
+Backup Config:
+Compress Process Number:  0
+Tag:                      daily
+Timeout:                  18000
+Cluster Name:               test-cluster
+Destination:
+Local:
+Path:   /home/tigergraph/backup
+Storage:  local
+Schedule:     * * * * *
+Status:
+Conditions:
+Last Transition Time:  2022-12-20T02:42:01Z
+Message:               Backup job is active
+Reason:                BackupActive
+Status:                True
+Type:                  Active
+Job Counter:
+Successful Jobs:     1
+Last Schedule Time:    2022-12-20T02:42:00Z
+Last Successful Time:  2022-12-20T02:41:11Z
+
+Events:
+Type    Reason                   Age                From                      Message
+----    ------                   ----               ----                      -------
+Normal  Backup schedule created  2m1s               TigerGraphBackupSchedule  Create a new backup schedule success.
+Normal  Backup job succeed       60s                TigerGraphBackupSchedule  Last scheduled job succeed
+Normal  Backup job created       10s (x2 over 71s)  TigerGraphBackupSchedule  Schedule a new backup job
+----
+
+The Events section tells you whether the scheduled job was successful.
+
+== Pause or resume a backup schedule
+
+To pause a running backup schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule pause --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+The next backup job will not be scheduled until the schedule is resumed.
+
+To resume a paused schedule:
+
+[source.wrap, console]
+----
+kubectl tg backup-schedule resume --cluster-name test-cluster -n tigergraph \
+--tag localdaily
+----
+
+
+== Backup controls
+
+There are three options to support different implementations of backups:
+
+  --max-retry :                 set max times of retry for each backup
+  --max-backup-file :           set the max number of files you want to retain
+  --max-reserved-day :          set the max number of days you want to retain these backups
+
+You can use them to control the backup job and manage backup files.
+
+=== Maximum number of backup files
+
+Over time, backup files will accumulate and take up disk space.
+Set `--max-backup-file` and `--max-reserved-day`, and the `TigerGraphBackupSchedule` process will help you delete outdated backups automatically based on the strategy you set.
+
+Assume your backup schedule has `--tag daily`.
+If you set `--max-backup-file` to `10`, when a scheduled backup process is completed, a cleaning process runs to remove outdated backups also tagged `daily`.
+Ten backups tagged `daily` are retained, while backups not tagged `daily` will not be affected.
```

Comment:
```suggestion
The ten newest backups tagged `daily` are retained, while backups not tagged `daily` will not be affected.
```

Reply:
I don't know for a fact that the command ranks them my date, but I certainly hope that it does.

Reply:
Yes, the command ranks all backups by date